src/Podium.php

```diff
@@ -185,6 +194,10 @@ public function init()
         } else {
             $this->podiumComponent->registerConsoleComponents();
         }
+        if (!is_string($this->slugGenerator) || $this->slugGenerator != '') {

```

Comment:
if (!is_string($this->slugGenerator)) {

Put this inside `if (Yii::$app instanceof WebApplication)` condition.